worlds/sm64ex/docs/setup_en.md

```diff
@@ -59,6 +59,56 @@ If everything worked out, you will see a textbox informing you the connection ha
 To play offline, first generate a seed on the game's settings page.
 Create a room and download the `.apsm64ex` file, and start the game with the `--sm64ap_file "path/to/FileName"` launch argument.
 
+# Using Batch Files to play offline and MultiWorld games
+
+As an alternative to launching the game with sm64pclauncher, it is also possible to launch the completed build with the use of Windows batch files. This has the added benefit of streamlining the join process so that manual editing of connection info is not needed for each new game. However, you'll need to be somewhat comfortable with creating and using batch files.
+
+IMPORTANT NOTE: The remainder of this section uses copy-and-paste code that assumes you're using the US version. If you instead use the Japanese version, you'll need to edit the EXE name accordingly by changing "sm64.us.f3dex2e.exe" to "sm64.jn.f3dex2e.exe".
+
+### Making an offline.bat for launching offline patch files
+
+Open Notepad. Paste in the following text: `start sm64.us.f3dex2e.exe --sm64ap_file %1 --skip-intro`
```

Comment:
```suggestion
Open Notepad. Paste in the following text: `start sm64.us.f3dex2e.exe --sm64ap_file %1`
```
Don't include options with the batch file; this would override the config file settings for `skip_intro`.